typings/telegram-types.d.ts

```diff
@@ -65,9 +65,9 @@ export type DiceEmoji = '🎲' | '🎯' | '🏀'
  */
 export type InputFileVideoNote = Exclude<TT.InputFile, TT.InputFileByURL>
 
-type MakeExtra<
+export type MakeExtra<
   M extends keyof TT.Telegram, // method name
-  P extends keyof Omit<TT.Opts<M>, 'chat_id'> // fields to skip
+  P extends keyof Omit<TT.Opts<M>, 'chat_id'> = never // fields to skip
 > = Omit<TT.Opts<M>, 'chat_id' | P>
```

Comment:
How about improving the param names instead of commenting?
```suggestion
export type MakeExtra<
  MethodName extends keyof TT.Telegram,
  SkipFields extends keyof Omit<TT.Opts<MethodName>, 'chat_id'> = never
> = Omit<TT.Opts<MethodName>, 'chat_id' | SkipFields>
```

Reply:
I'm hesitant towards using long names as type parameters, but I added proper docs now